nautobot/docs/release-notes/version-1.3.md

```diff
@@ -52,7 +52,7 @@ As Python 3.6 has reached end-of-life, the default Docker images published for t
 - [#863](https://github.com/nautobot/nautobot/issues/863) - Added the ability to hide a job in the UI by setting `hidden = True` in the Job's inner `Meta` class
 - [#881](https://github.com/nautobot/nautobot/issues/881) - Improved the UX of the main Jobs by adding accordion style interface that can collapse/expand jobs provided by each module
 - [#885](https://github.com/nautobot/nautobot/issues/885) - Added the ability to define a `soft_time_limit` and `time_limit` in seconds as attributes of a Job's `Meta`.
-- [#898](https://github.com/nautobot/nautobot/issues/898) - Added support for moving a CustomField, Relationship or ComputedField from the main tab of an object's detail page in the UI to the "Advanced" tab.
+- [#1001](https://github.com/nautobot/nautobot/issues/1001) - Added Job database model and associated functionality.
```

Comment:
Looks like we lost your release-note addition in the merge.

Reply:
D'oh! Adding back in now!